# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: netwerk/test/unit/test_cache_jar.js
# Commit: 74f99033251c
# Full Hash: 74f99033251cd6c36eb6b0b5a4713eb6b5c793fa
# Author: Jonathan Kingston <jkt@mozilla.com>
# Date: 2019-02-13 09:48:26
# Regressor Bug: 1520868
# File Overlap Count: 1
# Description:
#   Bug 1520868 - Replacing AsyncOpen2 with AsyncOpen always r=valentin
#   
#   Replacing js and text occurences of asyncOpen2
#   Replacing open2 with open
#   
# ==============================================================================

diff -r 3378fd5f4f22 -r 74f99033251c netwerk/test/unit/test_cache_jar.js
--- a/netwerk/test/unit/test_cache_jar.js	Tue Feb 12 17:58:19 2019 +0100
+++ b/netwerk/test/unit/test_cache_jar.js	Tue Feb 12 16:08:25 2019 +0000
@@ -101,7 +101,7 @@
 function test_channel(appId, inIsolatedMozBrowser, userContextId, expected) {
   return new Promise(resolve => {
     var chan = makeChan(URL, appId, inIsolatedMozBrowser, userContextId);
-    chan.asyncOpen2(new ChannelListener(doneFirstLoad.bind(null, resolve), expected));
+    chan.asyncOpen(new ChannelListener(doneFirstLoad.bind(null, resolve), expected));
   });
 }
 
@@ -109,7 +109,7 @@
   // Load it again, make sure it hits the cache
   var oa = req.loadInfo.originAttributes;
   var chan = makeChan(URL, oa.appId, oa.isInIsolatedMozBrowserElement, oa.userContextId);
-  chan.asyncOpen2(new ChannelListener(doneSecondLoad.bind(null, resolve), expected));
+  chan.asyncOpen(new ChannelListener(doneSecondLoad.bind(null, resolve), expected));
 }
 
 function doneSecondLoad(resolve, req, buffer, expected) {